src/IBM-pibs.xml

```diff
@@ -0,0 +1,20 @@
+<SPDX name="IBM PowerPC Initialization and Boot Software" identifier="IBM-pibs" osi-approved="false">
+  <urls>
+    <url>http://git.denx.de/?p=u-boot.git;a=blob;f=arch/powerpc/cpu/ppc4xx/miiphy.c;h=297155fdafa064b955e53e9832de93bfb0cfb85b;hb=9fab4bf4cc077c21e43941866f3f2c196f28670d</url>
+  </urls>
+  <license>
+    <body>
+      <p>This source code has been made available to you by IBM on an AS-IS basis. Anyone receiving this source is
+         licensed under IBM copyrights to use it in any way he or she deems fit, including copying it,
+         modifying it, compiling it, and redistributing it either with or without modifications. No license
+         under IBM patents or patent applications is to be implied by the copyright license.</p>
+      <p>Any user of this software should understand that IBM cannot provide technical support for this software
+         and will not be responsible for any consequences resulting from the use of this software.</p>
+      <p>Any person who transfers this source code or any derivative work must include the IBM copyright notice,
+         this paragraph, and the preceding two paragraphs in the transferred software.</p>
+      <p>COPYRIGHT I B M CORPORATION 2002 
```

Comment:
I think this is another "vanity" licenses where a name appears throughout, so not marking up copyright is okay (and we didn't decide to mark it up when we added it.